docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -671,6 +671,11 @@ CREATE TABLE example.default.new_table (id BIGINT, address VARCHAR);
 The Delta Lake connector also supports creating tables using the {doc}`CREATE
 TABLE AS </sql/create-table-as>` syntax.
 
+#### Schema evolution
+
+Delta Lake supports schema evolution, with safe column add, drop, and
```

Comment:
Delta Lake format supports schema evolution on nested fields, right? We could rephrase to "The connector supports ..." to clarify this is our connector's limitation.

Reply:
I tried out schema evolution operations for nested fields and got exceptions:

```
trino> alter table  delta.tiny.t3 drop column parent.child2;
Query 20250611_074645_00015_2q48r failed: This connector does not support dropping fields
```

```
trino> alter table delta.tiny.t2 rename column parent.child to newchild;
Query 20250611_073950_00010_2q48r failed: This connector does not support renaming fields
```